sporkfish/searcher/searcher.py

```diff
@@ -27,12 +20,8 @@ def __init__(self, config: SearcherConfig = SearcherConfig()) -> None:
         """
         Initialize the Searcher instance with mutable statistics.
 
-        :param evaluator: The chess board evaluator.
-        :type evaluator: evaluator.Evaluator
-        :param max_depth: The maximum search depth for the minimax algorithm. Default is 5.
-        :type max_depth: int
         :param config: Config to use for searching.
-        :type mode: SearcherConfig
+        :type config: SearcherConfig
```

Comment:
Good catch, in anticipation of other types of config, suggest rename to `searcher_config`?